src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -137,14 +138,22 @@ export class HTMLMarkdownConverter {
 		this.turndownService.addRule('p', {
 			filter: 'p',
 			replacement: function (content, node) {
+				let isAnchorElement: boolean = false;
 				node.childNodes.forEach(c => {
 					if (c.nodeType === Node.TEXT_NODE) {
 						c.nodeValue = escapeAngleBrackets(c.textContent);
 					} else if (c.nodeType === Node.ELEMENT_NODE) {
 						c.innerText = escapeAngleBrackets(c.textContent);
+						if (c.nodeName === 'A') {
```

Comment:
This is always going to be capitalized?

Reply:
Yes, as far as the turndown GitHub documentation I looked at all node names that are capitalized.